be/src/exec/vectorized/hdfs_scanner.cpp

```diff
@@ -218,7 +221,8 @@ void HdfsScanner::update_counter() {
 
 void HdfsScannerContext::set_columns_from_file(const std::unordered_set<std::string>& names) {
     for (auto& column : materialized_columns) {
-        if (names.find(column.col_name) == names.end()) {
+        auto col_name = case_sensitive ? column.col_name : boost::algorithm::to_lower_copy(column.col_name);
```

Comment:
> have you considered to add a method to return column name with `case_sensitive` as flag? much simpler I guess.

Because it appears in different classes, so I didn't add a method.  You're right, it can be much simpler, Let me add one.